ledger/ledger-runner-common/src/test/scala/com/daml/ledger/runner/common/PureConfigReaderWriterSpec.scala

```diff
@@ -189,6 +196,14 @@ class PureConfigReaderWriterSpec
     )
   }
 
+  it should "not support unknown keys" in {
+    convert(
+      jwtTimestampLeewayConfigConvert,
+      "some-crap=yes\n" + validJwtTimestampLeewayValue,
```

Comment:
I'd suggest the key `some-unknown-key` instead ; )

Reply:
agreed